downstream/titles/release-notes/topics/aap-24.adoc

```diff
@@ -86,37 +110,10 @@ The following bugs were fixed in this release of {PlatformNameShort}:
 
 * Added a FQDN check for the {HubName} hostname required to run the Skopeo commands.
 
-* Fixed an issue such that the front end URL for Red Hat Single Sign On (SSO) is now properly configured after you specify the `sso_redirect_host` variable.
+* The front end URL for Red Hat Single Sign On (SSO) is now properly configured after you specify the `sso_redirect_host` variable.
```

Comment:
This list item doesn't conform with grammatical parallelism with rest of the list points. Looks like almost all start with a verb in past tense, so the earlier construction was better suited.